scripts/build_docs.sh

```diff
@@ -29,22 +35,16 @@ template > docbuild/lakefile.toml
 # Substitute the toolchain from lean-toolchain into docbuild/lakefile.toml
 sed -i s/TOOLCHAIN/`grep -oP 'v4\..*' lean-toolchain`/ docbuild/lakefile.toml
 
-# Fetch the docs cache if it exists
-mkdir -p docs/docs
-mv docs/docs docbuild/.lake/build/doc
-
 # Initialise docbuild as a Lean project
 cd docbuild
-MATHLIB_NO_CACHE_ON_UPDATE=1 # Disable an error message due to a non-blocking bug. See Zulip
-~/.elan/bin/lake update FLT
-~/.elan/bin/lake exe cache get
+
+# Disable an error message due to a non-blocking bug. See Zulip
+MATHLIB_NO_CACHE_ON_UPDATE=1 ~/.elan/bin/lake update FLT
```

Comment:
Just echoing Ruben's comment on Zulip -- is there a reason we're not getting cache any more?

Reply:
Yes, because it's unnecessary since `lake update` already does that.